docs/styles.less

```diff
@@ -1598,4 +1598,8 @@ Spark table documentation styles
     max-width: 350px;
     border-radius: 0;
     z-index: 1;
+}
+
+.icon-code-snippets {
```

Comment:
Could we possibly alter this class name to indicate that this class is for the popover as I initially assumed this was for the code snippets that displays the code needed to use an icon

Reply:
renamed